src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/ResultItem.tsx

```diff
@@ -679,9 +701,16 @@ const tripSummary = (
     ),
   );
 
-  const requiresBookingText = tripPattern.bookingRequirement?.requiresBooking
-    ? t(TripSearchTexts.results.resultItem.footer.requiresBooking)
-    : undefined;
+  const bookingsRequiredCount =
+    getTripPatternBookingsRequiredCount(tripPattern);
```

Comment:
Seems like this function is only used here, consider move it to an internal function, instead of in `utils/`.

Reply:
We agreed to have it on utils for future reusability.